doc/api/errors.md

```diff
@@ -586,6 +586,10 @@ Used when `Console` is instantiated without `stdout` stream or when `stdout` or
 
 Used when the native call from `process.cpuUsage` cannot be processed properly.
 
+<a id="ERR_DEPRECATED_METHOD"></a>
+### ERR_DEPRECATED_METHOD
```

Comment:
To avoid confusion, perhaps `ERR_NO_LONGER_SUPPORTED` would be better here. The method itself is not deprecated, only one particular way of calling it.